src/clusterfuzz/_internal/system/environment.py

```diff
@@ -1141,3 +1141,36 @@ def wrapper(*args, **kwargs):
     return None
 
   return wrapper
+
+
+def is_testcase_deprecated(platform_id=None):
+  """Whether or not the device or branch is deprecated."""
+  if not is_android(platform_id.upper()):
+    return False
+
+  if is_android_cuttlefish(platform_id.upper()):
+    return False
+
+  if is_android(platform_id.upper()):
+    # FIXME: Handle this import in a cleaner way.
+    from clusterfuzz._internal.platforms import android
+
+    return android.util.is_testcase_deprecated(platform_id)
+
+  return False
+
+def can_testcase_run_on_platform(testcase_platform_id, current_platform_id):
+  """Whether or not the testcase can run on the current platform."""
+  if not is_android(testcase_platform_id.upper()) or not is_android(current_platform_id.upper()):
+    return False
+
+  if is_android_cuttlefish(testcase_platform_id.upper()) and is_android_cuttlefish(current_platform_id.upper()):
+    return True
+
+  if is_android(testcase_platform_id.upper()):
+    # FIXME: Handle this import in a cleaner way.
```

Comment:
I assume you noticed this import done similarly elsewhere in the file, but is it possible to just import this at the top of file in the standard way?

Reply:
Yes.  I took reference from line 215 in the same file - https://github.com/google/clusterfuzz/pull/3583/files#diff-ec10fe9c1dd83b9edcd526a0d57d8580bb8d764d4eab9c3c3e528d041e7a6487L215

I was not sure if it is possible to import this the standard way and did not want to break anything. Hence followed the existing method.

Reply:
Sorry I'm not sure this link shows this. Can you just import it at the top of the file instead.

Reply:
Done. Moved the import to the top and removed other duplicate imports